public/js/track.js

```diff
@@ -235,6 +247,7 @@ function addPlaylistEventListeners() {
 
     createPlaylistButton.addEventListener("click", async function () {
         let isConfirmPlaylistChecked = confirmPlaylistInput.checked;
+        $("#errorAlert").html("").hide();
```

Comment:
Does `html("")` set the html as empty?

Reply:
It sets it to an empty string so that the next time an alert pops up again, there isn't an accumulation of text!